tests/conftest.py

```diff
@@ -14,10 +14,16 @@
     PrivateFormat,
     load_pem_private_key,
 )
-from typing_extensions import Literal
 from uvicorn.config import Config
 from uvicorn.main import Server
 
+# `typing.Literal` exists from Python 3.8 onwards.
+# For <=3.7 we require the `typing_extensions` package for a backported version.
+if sys.version_info >= (3, 8):
+    from typing import Literal
```

Comment:
so the comment is also removed when pyupgrade is run:

```suggestion
if sys.version_info >= (3, 8):
    # `typing.Literal` exists from Python 3.8 onwards.
    # For <=3.7 we require the `typing_extensions` package for a backported version.
    from typing import Literal
```